### PR TITLE
Use https endpoints, avoid double slashes

### PR DESCRIPTION
**Pull request by andreiko**

### CODE REVIEW

File: lib/client.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 require('object.assign').shim();
 var request = require('request');
 
-var defaultBaseUrl = 'http://api.hubapi.com/';
+var defaultBaseUrl = 'https://api.hubapi.com/';
```
Comment (ynguyensegment) [inline lib/client.js]: Could you just remove the dash here and keep the rest the same instead?

Comment (deankarn) [reply lib/client.js]: I agree, I think this is the better option for clarity

Comment (cdemwell) [reply lib/client.js]: Agreed, changes in incidents should be kept to a minimum because we're acting under pressure.

**Review by deankarn — APPROVED** [review]

File: lib/client.js (hunk 1 above)
Comment (andreiko) [reply lib/client.js]: updated

**Review by deankarn — APPROVED** [review]

**Pull request merged**